srcpkgs/r2ghidra-dec/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'r2ghidra-dec'
+_cutterversion=1.11.1
+_ghidracommit=6c10f36f06468f866188cccf960c019779fb9028
+_pugixmlcommit=fdf02957533645069bbea7cd2ed85a3014b42cf3
+pkgname=r2ghidra-dec
+version=4.5.0
+revision=1
+archs="x86_64"
```

Comment:
Is this really only for `x86_64`, or is it `nocross`?

Reply:
`ghidra` seems to be `x86_64*`, perhaps it's the same issue? @abenson is there some comment that can be left on the template?

Reply:
Well this should then be at least `x86_64*` then probably.